internal/server/v1/event/golden/event_collection/CycloneEvent_Earth_201005.json

```diff
@@ -1,56 +1,6 @@
-            ]
-          },
-          "maxClassification": {
-            "vals": [
-              "TropicalStorm"
-            ]
-          },
-          "maxWindSpeed": {
-            "vals": [
-              "Knot45"
-            ]
-          },
-          "minPressure": {
-            "vals": [
-              "Millibar989"
-            ]
-          },
-          "name": {
-            "vals": [
-              "Bandu"
-            ]
-          }
-        },
-        "dcid": "cyclone/ibtracs_2010136N09057",
```

Comment:
@ajaits is this deletion expected?

Reply:
No, the cyclone events have had no script change. 

Does adding geoJson for ocean basins affect IBTrACS? 

Is there a way to query for all cyclone events that have been deleted to see if only the ones in the oceans that got the oceanBasin boundaries are missing?

Reply:
Not sure how adding geoJson for ocean basis affects IBTrACS.  We would need to run the import differ between two caches I think?